listsofwikis/mediawiki/shoutwiki-spider.py

```diff
@@ -1,55 +1,73 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-# Copyright (C) 2014 WikiTeam developers
+# Copyright (C) 2021 Simon Liu
```

Comment:
Doesn't matter much, but: 2021?

Reply:
oops i guess i'm still mentally living in 2021 🥲